submissions/YuliiaHordiichuk/friends-app/app.js

```diff
@@ -0,0 +1,130 @@
+const randomUsersUrl = 'https://randomuser.me/api/?results=36&inc=gender,name,email,phone,picture,dob';  
+
+let users; 
+let searchBy = ''; 
+let sortBy = ''; 
+let filterBy = 'all'; 
+ 
+function getData() {
+    fetch(randomUsersUrl)
+        .then(answer => answer.json())
```

Comment:
https://www.tjvantoll.com/2015/09/13/fetch-and-errors/

Reply:
Thanks, very helpful!